models/tokenprice/tokenprices.sql

```diff
@@ -0,0 +1,27 @@
+{{
+    config(
+        materialized='incremental',
+        unique_key="block_date||'-'||token_address",
+        tags=['core', 'defi', 'tokenprice'],
+        cluster_by=['block_date', 'token_address']
+        )
+}}
+
+with 
+stage as (
+  select * from {{ ref('stg_tokenprice_from_swaps')}}
+),
+
+final as (
+  SELECT
+    block_date,
```

Comment:
```suggestion
    key,
    block_date,
```
in conjunction with the unique_key being added to the incremental